lib/src/file-input/FileInput.tsx

```diff
@@ -285,7 +284,7 @@ const DxcFileInput = React.forwardRef<RefType, FileInputPropsType>(
   }
 );
 
-const FileInputContainer = styled.div`
+const FileInputContainer = styled.div<{ margin: FileInputPropsType["margin"]; name: FileInputPropsType["name"] }>`
```

Comment:
Yes, it it not used in the styled component but necessary for the case of a file input inside a form. I will remove it from the styled component typing!

Reply:
But that's an error. `name` attribute does not exist in a `<div>` element so it won't do anything inside a form. Since we remove it from the input to avoid the component from sending wrong values when submitting, I think we should remove the prop. It doesn't do anything at this point.

If you pass a HTML markup validator it will trigger this as an error:
![image](https://user-images.githubusercontent.com/44321109/233622286-eaa3cc5f-2afa-4f9c-8f97-79b9d6062688.png)

Reply:
Code from the example above:

![image](https://user-images.githubusercontent.com/44321109/233622477-d747a5b1-c8d9-40b1-a94c-b1897d7ac645.png)

Reply:
Totally agree!!